packages/theme-base/src/themes/sap_fiori_3/base-parameters.less

```diff
@@ -256,6 +256,12 @@
 @sapTile_IconColor: contrast(@sapTile_Background, lighten(@sapShellColor ,20), @sapContent_ContrastIconColor, @sapContent_ContrastTextThreshold);
 @sapToolbar_SeparatorColor: darken(@sapBaseColor, 15);
 
-// custom added params as no base ("sap" prefix) params equivalent exist
+
 @sapToggleButton_Pressed_HoverBackground: lighten(@sapSelectedColor, 3);
-@sapToggleButton_Pressed_HoverBorderColor: @sapToggleButton_Pressed_HoverBackground;
\ No newline at end of file
+@sapToggleButton_Pressed_HoverBorderColor: @sapToggleButton_Pressed_HoverBackground;
+
+
+@sapTimePicker_ItemBorderColor: @sapList_Background;
+@sapTimePicker_ItemColor: darken(@sapList_Background, 3);
```

Comment:
We already have --sapList_Background_Darken3,  --sapList_Background_Darken10.
You can add @sapList_Background_Darken6, that is missing in:

theme-base/src/themes/base/component-derived-colors.less
theme-base/src/themes/base/component-derived-colors-vars.less

Reply:
done